packages/lab/src/Flow/types/flow.ts

```diff
@@ -8,57 +8,40 @@ import { HvColorAny } from "@hitachivantara/uikit-styles";
 
 // Node types
 
-type NodeExtras<GroupId extends keyof any = string, NodeData = any> = {
-  meta?: HvFlowNodeTypeMeta<GroupId, NodeData>;
-};
-
 /** HvFlowNode component type. @extends React.FC */
-export interface HvFlowNodeFC<
-  GroupId extends keyof any = string,
-  NodeData = any
-> extends FC<NodeProps<NodeData>>,
-    NodeExtras<GroupId, NodeData> {}
-
-export interface HvFlowNodeComponentClass<
-  GroupId extends keyof any = string,
-  NodeData = any
-> extends ComponentClass<NodeProps>,
-    NodeExtras<GroupId, NodeData> {}
-
-export type HvFlowNodeComponentType<
-  GroupId extends keyof any = string,
-  NodeData = any
-> =
-  | HvFlowNodeComponentClass<GroupId, NodeData>
-  | HvFlowNodeFC<GroupId, NodeData>;
-
-export type HvFlowNodeTypes<
-  GroupId extends keyof any = string,
-  NodeData = any
-> = Record<string, HvFlowNodeComponentType<GroupId, NodeData>>;
+export interface HvFlowNodeFC<NodeData = any> extends FC<NodeProps<NodeData>> {}
+
+export interface HvFlowNodeComponentClass extends ComponentClass<NodeProps> {}
+
+export type HvFlowNodeComponentType<NodeData = any> =
+  | HvFlowNodeComponentClass
+  | HvFlowNodeFC<NodeData>;
+
+export type HvFlowNodeTypes<NodeData = any> = Record<
+  string,
+  HvFlowNodeComponentType<NodeData>
+>;
+
+export interface HvFlowGroupItem {
+  /** Node type */
+  type: string;
+  label: string;
+  data?: unknown;
+}
 
 /** Node groups */
 export interface HvFlowNodeGroup {
   label: string;
   description?: string;
   color?: HvColorAny;
   icon?: React.ReactNode;
+  items?: Record<string, HvFlowGroupItem>;
```

Comment:
why not
```suggestion
  items?: HvFlowGroupItem[];
```

aren't the object's `key` and `.id` the same?

Reply:
I added the key to have a unique id for the item